custom/conf/app.example.ini

```diff
@@ -1485,6 +1485,11 @@ LEVEL = Info
 ;; - manage_ssh_keys: a user cannot configure ssh keys
 ;; - manage_gpg_keys: a user cannot configure gpg keys
 ;USER_DISABLED_FEATURES =
+;; Disabled features if the user has an external login type. Can be the same options as for USER_DISABLED_FEATURES.
+;; - deletion: a user cannot delete their own account
+;; - manage_ssh_keys: a user cannot configure ssh keys
+;; - manage_gpg_keys: a user cannot configure gpg keys
+;;EXTERNAL_USER_DISABLE_FEATURES =
```

Comment:
What is the syntax of this? Comma-separated? If so, start sentence with `Comma-seprated list of ...`